docs/quick_start.md

```diff
@@ -31,7 +31,7 @@ mkdir <new_dir> && cd <new_dir>
 **2. Start a git repository**
 
 ```
-git init && git remote add -f https://github.com/triton-inference-server/model_analyzer.git
+git init && git remote add -f origin https://github.com/triton-inference-server/model_analyzer.git
```

Comment:
is origin not the default?

Reply:
I just tried it out and it failed for me without origin

![image](https://user-images.githubusercontent.com/50968584/173632876-e4b3fc48-9e49-42f7-ab5f-f0c47cd55671.png)

Reply:
thanks for checking Tim.

Reply:
No. We need to specify the name for url.